### DISCUSSION TITLE
Possibility to move idle vehicle out of the way without parking positions

### DISCUSSION THREAD

**Discussion by Joeriubink** [openTCS/opentcs#106] [Q&A] [answered]
![image](https://github.com/openTCS/opentcs/assets/58429440/ca37aa00-3d62-4f1e-a37b-6769017e3c80)

As seen in the Image, Vehicle-0001 has a transport order from Delivery 1 to Unit 1. However, Vehicle 2 is standing in the way, being idle. This causes vehicle 1 to be stuck as long as there is no new transportation task which will be assinged to vehicle 2 and therefore moves.
Note: I do not have specified parking positions. I just want to know if it is possible to automatically create a task for Vehicle-0002 to move out of the way (so for example move to "Section 2")

**Comment by swltr** [comment, marked as answer]
@Joeriubink No - at least there is no built-in functionality for that. (One solution/workaround would be to integrate a custom kernel extension that periodically scans for idle vehicles that are obstacles for other vehicles, and create transport orders for them to move them out of the way.)
